tests/integration/009_team/main.tf

```diff
@@ -22,4 +22,24 @@ output "team_resource_description" {
 
 output "team_data_description" {
   value = data.env0_team.team_data.description
-}
\ No newline at end of file
+}
+
+resource "env0_custom_role" "custom_role1" {
+  name = "custom-role-${random_string.random.result}"
+  permissions = [
+    "CREATE_PROJECT"
+  ]
+}
+
+resource "env0_custom_role" "custom_role2" {
+  name = "custom-role-${random_string.random.result}2"
+  permissions = [
+    "RUN_PLAN"
+  ]
+}
+
+resource "env0_team_organization_assignment" "organization_role_assignment" {
+  team_id = env0_team.team_resource.id
+  role_id = var.second_run ? env0_custom_role.custom_role1.id : env0_custom_role.custom_role2.id
```

Comment:
What's the purpose of this `var.second_run`? to show how to change assignments upon 2nd run?

Reply:
yes - this is how updates are tested.
first run is create... second run is an update (covers more use-cases).